modules/cms/console/CreateTheme.php

```diff
@@ -77,6 +78,70 @@ class CreateTheme extends GeneratorCommand
         ],
     ];
 
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        // Originale Theme-Erstellung
+        $result = parent::handle();
+
+        // Theme-Code definieren
+        $themeCode = 'theme-' . $this->getNameInput();
+
+        // Wenn Tailwind gewählt wurde
+        if ($this->argument('scaffold') === 'tailwind') {
+            $this->info('Tailwind CSS wurde gewählt. Installiere notwendige Abhängigkeiten...');
+
+            try {
+                // Mix installieren mit automatischen Antworten
+                $descriptorspec = array(
+                    0 => array("pipe", "r"),  // stdin
+                    1 => array("pipe", "w"),  // stdout
+                    2 => array("pipe", "w")   // stderr
+                );
+                
+                $process = proc_open('php artisan mix:install', $descriptorspec, $pipes, base_path());
+                
+                if (is_resource($process)) {
+                    // Alle "yes" Antworten senden
+                    fwrite($pipes[0], "yes\n");
+                    fwrite($pipes[0], "yes\n");
+                    fwrite($pipes[0], "yes\n");
+                    fwrite($pipes[0], "yes\n");
+                    fclose($pipes[0]);
+                    
+                    // Output lesen
+                    $output = stream_get_contents($pipes[1]);
+                    $this->info($output);
+                    
+                    // Fehler lesen
+                    $errors = stream_get_contents($pipes[2]);
+                    fclose($pipes[1]);
+                    fclose($pipes[2]);
+                    
+                    proc_close($process);
+                }
```

Comment:
Is this approach used instead of Artisan::call() because the mix:install command is interactive? Could we instead add support for a `--force` option on `mix:install`? @jaxwilko any thoughts?

Reply:
I would be better to have it as an option rather than assuming people want to install mix/vite right at this point.

Reply:
@josephcrowell what situation can you think of when someone scaffolds a tailwind based theme and doesn't want to have the asset compilation working on it?

Reply:
> Is this approach used instead of Artisan::call() because the mix:install command is interactive? Could we instead add support for a `--force` option on `mix:install`? @jaxwilko any thoughts?

on my local machine ite mix:install is automaticlly not interactive   

@josephcrowell i didn't know a situation to use scaffolds tailwind without this compilation but you can "create" a exta button for tailwind without mix - this is easy   

Today I will finish scaffolding tailwind with & vue

Reply:
They were curious about the theme and decided to install without realizing about the Mix/Vite requirement or they were going to install other things before running the Mix/Vite commands. Not a huge deal though I guess.

Reply:
You could run the command the same way it's done in the tests. Laravel creates a mock output which simulates questions being asked and allows you to answer them:
```php
(new \Illuminate\Testing\PendingCommand(new \System\Tests\Bootstrap\TestCase(), $this->getLaravel(), 'mix:install', []))
    ->expectsQuestion('Detected winter.blocks (plugins/winter/blocks), should it be added to your package.json?', true)
    ->expectsQuestion('Detected winter.sentry (plugins/winter/sentry), should it be added to your package.json?', true);
```
However this is really difficult to work with as depending on the packages installed by the user you may get different results...

The other option is to call `mix:install --no-interaction` which would just auto accept all the options which I think is what you want:
```php
Artisan::call('mix:install', [
    '--no-interaction' => true
]);
```